crates/astria-bridge-contracts/src/lib.rs

```diff
@@ -723,9 +731,22 @@ fn timeout_in_5_min() -> u64 {
 
 #[cfg(test)]
 mod tests {
-    use super::max_timeout_height;
+    use super::*;
+
     #[test]
     fn max_timeout_height_does_not_panic() {
         max_timeout_height();
     }
+
+    #[test]
+    fn memo_snapshot() {
+        let event = Ics20WithdrawalFilter {
+            sender: "14cbd075f796969cf5dbd853da492d86071f97ed".parse().unwrap(),
+            amount: ethers::core::types::U256::one(),
+            destination_chain_address: "destination address".to_string(),
+            memo: "a memo".to_string(),
```

Comment:
```suggestion
memo: "<a third party memo>".to_string(),

Reply:
Done in [32b1522](https://github.com/astriaorg/astria/pull/1753/commits/32b15220e4e70a75cc134a150d1c0d3c9fdfd222).